tests/conftest.py

```diff
@@ -18,7 +18,7 @@
 from . import utils
 
 DOWNLOAD_DIR = os.path.abspath(os.path.join("tests", "data", "_tmp_download"))
-
+DATA_ROOT_DIR = os.path.join(os.path.dirname(__file__), "data")
```

Comment:
```suggestion
DATA_ROOT_DIR = os.path.join(os.path.dirname(__file__), "data")
DOWNLOAD_DIR = os.path.join(DATA_ROOT_DIR, "_tmp_download")
```
How about that? It would make the tests portable (the tests were making the assumption that pytest was run from the root of the repo, which is kind of bad and entirely my fault).

Reply:
totally, I did not touched it as I was not sure it was intentional or not !